webpack.config.js

```diff
@@ -269,7 +275,13 @@ function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv )
 			} ),
 			new webpack.NormalModuleReplacementPlugin( /^path$/, 'path-browserify' ),
 			new webpack.IgnorePlugin( /^props$/ ),
-			new MiniCssExtractPlugin(),
+			new MiniCssExtractPlugin( {
```

Comment:
What do you think about renaming this to `MiniCssExtractPluginWithRtl` so that it's clear it's `mini-css-extract-plugin-with-rtl`, not `mini-css-extract-plugin`?

Reply:
Whoop, I just saw your comment elsewhere https://github.com/Automattic/wp-calypso/pull/26820#discussion_r212592232

Reply:
FWIW, I kind of agree with @simison and @ockham on this one. If I see `MiniCssExtractPlugin`, I wouldn't infer it would be `with-rtl`, so that could cause some confusion.

Reply:
I disagree and implement in https://github.com/Automattic/wp-calypso/pull/26864/commits/394ca4061eaef4a2e3faf74947c3a6066de18611 😉